python/lsst/afw/image/exposureInfo.cc

```diff
@@ -45,6 +46,44 @@ namespace {
 
 using PyExposureInfo = py::class_<ExposureInfo, std::shared_ptr<ExposureInfo>>;
 
+// Template methods where we can use pybind11's overload resolution (T is input)
+template <class T>
+void declareGenericMethods(PyExposureInfo &cls) {
+    using Class = PyExposureInfo::type;
+    // py::overload_cast can't deduce return type... because Key is not pybind11-wrapped?
```

Comment:
FWIW, I've (rarely) seen the same problem with `overload_cast` on other methods (can't remember where, unfortunately) and went with the same workaround.  I also couldn't figure out what was wrong.